doc/index.rst

```diff
@@ -2,9 +2,15 @@
 Secure XGBoost Documentation
 ############################
 
-**Secure XGBoost** is a secure gradient boosted decision tree library based off the popular `XGBoost <https://github.com/dmlc/xgboost>`_ project that supports scalable, distributed, and efficient `gradient boosting <https://en.wikipedia.org/wiki/Gradient_boosting>`_. In addition to offering the efficiency, flexibility, and portability that vanilla XGBoost does to solve a variety of problems, Secure XGBoost enables secure collaborative learning by leveraging `hardware enclaves <https://inst.eecs.berkeley.edu/~cs261/fa18/slides/Hardware_Enclaves.pdf>`_ and `oblivious algorithms <https://www.usenix.org/system/files/conference/usenixsecurity16/sec16_paper_ohrimenko.pdf>`_. This project allows multiple parties to each share their sensitive data to perform joint computation *without revealing the contents of the data*.
 
-This project is currently under development as part of the broader `Multiparty Collaboration and Coopetition effort <https://github.com/mc2-project/mc2>`_ by the UC Berkeley `RISE Lab <https://rise.cs.berkeley.edu/>`_. 
+Secure XGBoost is a library for **privacy-preserving training and inference** of `XGBoost <https://github.com/dmlc/xgboost>`_ models.
+In particular, it facilitates secure collaborative learning -- where mutually distrustful data owners can jointly train a model on their data, but without revealing their data to each other.
+
+At its core, Secure XGBoost uses `secure hardware enclaves <https://en.wikipedia.org/wiki/Trusted_execution_environment>`_ (such as Intel SGX) to protect the data in use even in the presence of a hostile cloud environment.
```

Comment:
"to protect the data in use even in the presence" --> "to protect the data and computation even in the presence"?